giskard/cli.py

```diff
@@ -86,28 +89,29 @@ def start_command(host, port, is_server, is_daemon):
     - client: ML Worker acts as a client and should connect to a running Giskard instance
         by specifying this instance's host and port.
     """
-
-    _start_command(is_server, host, port, is_daemon)
+    _start_command(is_server, is_silent, host, port, is_daemon)
 
 
-def _start_command(is_server, host, port, is_daemon):
+def _start_command(is_server, is_silent, host, port, is_daemon):
+    token = click.prompt("Please enter an API Access Token")
```

Comment:
instead of prompting this value directly it's better to declare it as another cli option with a `prompt` property
like so
https://click.palletsprojects.com/en/8.1.x/options/#prompting